Resources/Prototypes/Catalog/Fills/Crates/emergency.yml

```diff
@@ -42,11 +43,29 @@
         amount: 3
       - id: OxygenTankFilled
         amount: 3
-      - id: EmergencyOxygenTankFilled
+      - id: NitrogenTankFilled
         amount: 3
       - id: ClothingOuterSuitEmergency
         amount: 3
 
+- type: entity
+  id: CrateEmergencyInternalsLarge
+  parent: CrateInternals
+  name: Emergency Suits (Large)
+  components:
+    - type: StorageFill
+      contents:
+        - id: ClothingMaskGas
+          amount: 12
+        - id: ClothingMaskBreath
+          amount: 12
+        - id: OxygenTankFilled
+          amount: 12
+        - id: NitrogenTankFilled
+          amount: 12
+        - id: ClothingOuterSuitEmergency
+          amount: 12
```

Comment:
That is an excessive amount of gas tanks for $2,000, that's at least 12 hours of breathing time.

Reply:
Appraisal tool says 1304 spacebucks so I'll make it 2600.

Reply:
Gasses are deliberately underpriced to avoid their sellcosts being exorbitant (given it is a free resource from atmosia) but doesn't mean they should be super cheap to buy (I am also aware of the other crate that needs a nerf).